root/static/scripts/common/components/CDTocLink.js

```diff
@@ -13,7 +13,11 @@ import entityHref from '../utility/entityHref.js';
 
 type Props = {
   +anchorPath?: string,
-  +cdToc: CDTocT,
+  +cdToc: {
+    +discid: string,
+    +entityType: 'cdtoc',
+    ...
```

Comment:
this probably doesn't need to be inexact?

Reply:
Isn't that needed if I also want to allow passing proper `CDTocT`s?

Reply:
Yeah, sorry, I forgot we still need to support those.